src/components/TabCloseButton.tsx

```diff
@@ -28,6 +28,7 @@ export function TabCloseButton({ isDirty, onClick }: CloseButtonProps) {
         className={cx('block', 'group-hover/tab-close-button:!block', {
           '!hidden': isDirty,
         })}
+        color="currentcolor"
```

Comment:
Why do we need this? Where have you seen this documentation?

Reply:
We need it to be able to set the color in the parent `div`, in `TabItem`. That's just a regular CSS keyword: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword